asmcomp/amd64/emit.mlp

```diff
@@ -911,21 +926,6 @@ let fundecl fundecl =
   D.label (emit_symbol fundecl.fun_name);
   emit_debug_info fundecl.fun_dbg;
   cfi_startproc ();
```

Comment:
Wouldn't it make sense for `cfi_startproc` to be part of what is emitted by `Lprelude`

Reply:
I will check this.

Reply:
I think this is ok as is, having thought about it this morning.  It matches up nicely with `cfi_endproc` at the moment; and it also seems in some sense correct that it's right at the top in the assembly output.